modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/dag/Registry.scala

```diff
@@ -0,0 +1,20 @@
+package org.tessellation.rosetta.server.model.dag
+
+import org.tessellation.ext.kryo.KryoRegistrationId
+import org.tessellation.schema.transaction.RewardTransaction
+
+import eu.timepit.refined.numeric.Interval
+
+object Registry {
+
+  import eu.timepit.refined.auto._
+
+  type RunnerKryoRegistrationIdRange = Interval.Closed[300, 399]
+
+  type RunnerKryoRegistrationId = KryoRegistrationId[RunnerKryoRegistrationIdRange]
+
+  val rosettaKryoRegistrar: Map[Class[_], RunnerKryoRegistrationId] = Map(
+    classOf[RewardTransaction] -> 389
+  )
```

Comment:
why it's needed?

Reply:
This is needed for the Rosetta endpoint implementations. `KryoSerializer` is used a lot there.

Reply:
I don't see any need for this for now. Also, RewardTransaction (if it's the one from global snapshot, it's already registered in kryo). You will be mostly mapping rosetta classes to our domain classes and these classes are already registered in kryo.

Reply:
This file is not currently needed and may not be needed anymore. It has been removed.